pkg/controller/sls/machineGroupBinding_controller.go

```diff
@@ -73,44 +71,16 @@ type machineGroupBindingConnector struct {
 func (c *machineGroupBindingConnector) Connect(ctx context.Context, mg resource.Managed) (managed.ExternalClient, error) {
 	cr, ok := mg.(*aliv1alpha1.MachineGroupBinding)
 	if !ok {
-		return nil, errors.New(errNotMachineGroupBinding)
+		return nil, errors.New(errNotLogtail)
```

Comment:
Is this correct? This appears to be the ExternalConnector for the machine group binding type.